sdk/tables/Azure.Data.Tables/tests/TableServiceClientLiveTests.cs

```diff
@@ -25,6 +25,12 @@ public class TableServiceClientLiveTests : TableServiceLiveTestsBase
         public TableServiceClientLiveTests(bool isAsync, TableEndpointType endpointType) : base(isAsync, endpointType /* To record tests, add this argument, RecordedTestMode.Record */)
         { }
 
+        [RecordedTest]
+        public async Task EnableTenantDiscovery()
+        {
+            (await client.QueryAsync<TableEntity>().ToEnumerableAsync()).ToList();
```

Comment:
It may be worth a comment to explain what this is used for, since the value is thrown away.

Reply:
I can add a comment, but it's just to ensure we can do an arbitrary operation without throwing - essentially, did auth work.